### PR TITLE
TER-59 named constraint when creating table with PK

### PR DESCRIPTION
**Pull request by jirkasemmler**
TER-59

query builder methods to add/drop pk
table utils: https://github.com/keboola/php-table-backend-utils/pull/91
TD driver : https://github.com/keboola/php-storage-driver-teradata/pull/51

### CODE REVIEW

File: src/Table/Teradata/TeradataTableQueryBuilder.php [hunk 1]
```diff
@@ -144,7 +145,8 @@ public function getCreateTableCommand(
 
         if ($primaryKeys !== []) {
             $columnsSql .= sprintf(
-                ",\nPRIMARY KEY (%s)",
+                ",\nCONSTRAINT %s PRIMARY KEY (%s)",
```
Comment (zajca) [inline src/Table/Teradata/TeradataTableQueryBuilder.php:148]:
```suggestion
                ',\nCONSTRAINT %s PRIMARY KEY (%s)',
```

Comment (zajca) [reply src/Table/Teradata/TeradataTableQueryBuilder.php:148]: blbé, že je to interface to by mohl byt parametr ten kbc_pk :/

**Review by zajca — APPROVED** [review]

File: src/Table/Teradata/TeradataTableQueryBuilder.php (hunk 1 above)
Comment (jirkasemmler) [reply src/Table/Teradata/TeradataTableQueryBuilder.php]: tam musi byt `"` jinak se ten `\n` nevyhodnoti

**Comment by jirkasemmler** [comment]
merguju, BQ je issue s rate limitem

**Pull request merged**